Various documentation improvements (#919)
tradingview/lightweight-charts: website/docs/price-scale.md

---
sidebar_position: 2
---

# Price scale

Price Scale (or price axis) is a vertical scale that mostly maps prices to coordinates and vice versa.
The rules of converting depend on a price scale mode, a height of the chart and visible part of the data.

![Price scales](/img/price-scales.png "Price scales")

By default, chart has 2 predefined price scales: `left` and `right`, and an unlimited number of overlay scales.

Only `left` and `right` price scales could be displayed on the chart, all overlay scales are hidden.

If you want to change `left` price scale, you need to use [`leftPriceScale`](/api/interfaces/ChartOptions#leftpricescale) option, to change `right` price scale use [`rightPriceScale`](/api/interfaces/ChartOptions#rightrricescale), to change default options for an overlay price scale use [`overlayPriceScales`](/api/interfaces/ChartOptions#overlaypricescales) option.

> Alternatively, you can use [`IChartApi.priceScale`](/api/interfaces/IChartApi#pricescale) method to get an API object of any price scale or [`ISeriesApi.priceScale`](/api/interfaces/ISeriesApi#pricescale) to get an API object of series' price scale (a price scale where the series is attached to).

Reviewer: ```suggestion
Alternatively, you can use [`IChartApi.priceScale`](/api/interfaces/IChartApi#pricescale) method to get an API object of any price scale or [`ISeriesApi.priceScale`](/api/interfaces/ISeriesApi#pricescale) to get an API object of series' price scale (the price scale that the series is attached to).
```
